Esmorga-Backend/esmorga-backend
# [MOB-134] BE - Unblock account after reach invalid login attempts

## Pull Request Checklist

- [x] **[MOB US' Link](https://mobilemakers.atlassian.net/browse/MOB-134)**
- [ ] **Add Tests** N/A

## Testing Guide
1. Register an user.
2. Active the user.
3. Make 5 wrong login attempts.
4. Check that the user is blocked with the correct credentials.
5. Change the date of the field `expireBlockedAt` as a past date.
6. Check that the user is active again and the `expireBlockedAt` is null.
